draft-ietf-rtcweb-jsep.xml

```diff
@@ -3670,7 +3705,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
       processing of the RTP packets is done at the RtpReceiver level.
       This includes using
       <xref target="I-D.ietf-mmusic-rid" /> to determine which RTP
-      streams depend on or repair other RTP streams.</t>
+      streams depend on or repair other RTP streams. If the RTP packet
+      did not have PT that matched any codec for this RtpReceiver, the
+      packet would be dropped. </t>
```

Comment:
I think you mean to say
"If the payload type in the RTP packet does not match any codec for this RtpReceiver, the packet will be dropped"